src/audiences/ui/components/rule.js

```diff
@@ -163,15 +173,29 @@ const RuleInput = props => {
 									} }
 								>
 									<option value="">{ __( 'Empty', 'altis-analytics' ) }</option>
-									{ data.map( datum => datum !== '' && (
+									{ topHits.length && (
```

Comment:
This should be an expression that results in a boolean, if `topHits.length` is 0 then "0" will be rendered:
```suggestion
									{ topHits.length > 0 && (
```

Reply:
I knew I should do that, just failed to remember why! :D will update.